projects/kit/components/stepper/stepper.component.ts

```diff
@@ -99,6 +98,10 @@ export class TuiStepperComponent {
         this.moveFocus(event.target, step);
     }
 
+    ngAfterViewChecked(): void {
```

Comment:
How many times that method calls?

Reply:
We need an early return if scrolling is unnecessary.